google-cloud-spanner/pom.xml

```diff
@@ -153,5 +159,25 @@
       <classifier>testlib</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-api</artifactId>
+      <exclusions>
+        <exclusion>
```

Comment:
Why is this required?

Reply:
I believe I was running into some issue with errorprone version conflicts. Let me try removing it and compiling again.